service/controller/resource/app/desired.go

```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/giantswarm/cluster-operator/v4/service/internal/releaseversion"
 )
 
+const (
+	DefaultAppExtraConfigPriority = 25
```

Comment:
There is a constants for these here: https://github.com/giantswarm/app/blob/master/pkg/values/extra_configs.go

Reply:
Updated this one.

Reply:
I thought I pushed an update some time ago to use the constant defined in the library 🤔